grammars/prql-lezer/test/arithmetics.txt

```diff
@@ -1,39 +1,39 @@
 # Plus
 
-10 + 10.5
+test 10 + 10.5
```

Comment:
Are all these `test`s supposed to be here? Sorry if I'm missing something...

Reply:
Yes, they are. They move the expression from the root into a call expression where "test" is the name of a hypothetical transform function and where `10 + 10.5` is the argument to that function.

Reply:
I see. Because we no longer allow a full query with just `10 + 10.5`, and these tests parse a full query?

Can we do a search / replace for `test` -> `filter`? `test` might be a bit confusing; it seems like it might be specifically related to the word `test` given it's a test.

It's up to you, but you might want to think about either parsing lots of small expressions (i.e. the existing code), or fewer bigger full queries. Because it's more difficult to manage if we have lots of small queries — we have all the overhead of `Query(Pipeline(CallExpression(Identifier,ArgList(` just to see what `2+2` gives us. We also have queries that aren't actually valid — there's no `from foo` — but are bigger than simple expressions.

But it's up to you, no pressure from me; I hope that's reasonable feedback.

Reply:
Well the PRQL compiler doesn't allow a full query with just `10 + 10.5` but the grammar does, but I would like change the grammar so that it does not.

Yes, we can change `test` to `filter`. The grammar doesn't care.
I used `test` since it "tests the expression". I could use `filter` but technically `filter 10 + 10.5` isn't a valid PRQL query, not that the grammar cares.

Reply:
@max-sixty I am awaiting the merge

Reply:
@max-sixty ping!

Reply:
@max-sixty I am looking to get this merged

Reply:
Sorry! I appreciate the patience.